Classes/UI/SparkConnectingProgressViewController.m

```diff
@@ -29,7 +29,7 @@
 NSInteger const kMaxRetriesConfigureAP = 5;
 NSInteger const kMaxRetriesConnectAP = 5;
 NSInteger const kMaxRetriesReachability = 5;
-NSInteger const kWaitForCloudConnectionTime = 1;
```

Comment:
I understand that Particle said 1, finger foods switched to 3.   And we might *keep* at 3.  Can you confirm what this  delay is in 1.0.9?

Reply:
Delay in 1.0.9 was 1 second